macOS/LocalPackages/SystemExtensionManager/README.md

```diff
@@ -1,5 +1,5 @@
-# SystemExtensionManager
+# NetworkExtensionManager
```

Comment:
Should we keep this the same as the old value since the package name itself hasn't changed? (Not that it matters really what value is put here tbh)